lib/cjs/generated/rum.d.ts

```diff
@@ -686,7 +686,7 @@ export declare type RumViewEvent = CommonProperties & {
         /**
          * The precondition that led to the creation of the session
          */
-        readonly start_reason?: 'app_start' | 'inactivity_timeout' | 'max_duration' | 'stop_api' | 'background_event';
+        readonly start_precondition?: 'app_launch' | 'inactivity_timeout' | 'max_duration' | 'explicit_stop' | 'background_event';
```

Comment:
is there backward compatibility on backend for the old values ?

Reply:
Old values were never deployed, they were merged yesterday.